src/UiPatternBase.php

```diff
@@ -109,6 +109,21 @@ public function getLibraries() {
   /**
    * {@inheritdoc}
    */
+  public function hasUse() {
+    $definition = $this->getPluginDefinition();
+    return isset($definition['use']) && !empty($definition['use']);
```

Comment:
Thanks, updated! About extra space: coding standards are checked by PHPCS already before every push running the following:

```
./vendor/bin/phpcs --standard=Drupal,DrupalPractice ./src ./tests/src ./modules --ignore=./modules/ui_patterns_test
```